src/datasources/asset/AssetDataSource.test.ts

```diff
@@ -208,19 +211,45 @@ const assetsResponseMock: AssetsResponse =
     "totalCount": 4
   }
 
-const assetUtilizationQueryMock: AssetQuery = {
-  type: AssetQueryType.Metadata,
+const assetsResponseMockCalibrationForecast: CalibrationForecastResponse = {
+  calibrationSummary: {
+    totalAssets: 10,
+    assetsApproachingCalibration: 6,
+    assetsPastDueCalibration: 4,
+  },
+  calibrationForecast: {
+    columns: [
+      { name: 'Month', values: ['2024-03', '2024-04', '2024-05', '2024-06', '2024-07', '2024-08', '2024-09', '2024-10', '2024-11', '2024-12'] },
+      { name: 'Approaching Calibration', values: [1, 2, 3, 4, 5, 6, 5, 4, 3, 2] },
+      { name: 'Past Due Calibration', values: [0, 1, 2, 3, 4, 4, 3, 2, 1, 0] },
+    ],
+  }
+}
+
+const assetUtilizationQueryMock: AssetMetadataQuery = {
+  queryKind: AssetQueryType.Metadata,
   workspace: '',
   refId: '',
-  minionIds: ['123'],
+  minionIds: ['123']
+}
+
+const queryMockCalibrationForecast: AssetCalibrationForecastQuery = {
```

Comment:
We are a bit inconsistent with naming here. My suggestion `calibrationForecastQueryMock`